samza-core/src/main/java/org/apache/samza/operators/spec/OperatorSpec.java

```diff
@@ -62,8 +69,17 @@
    * The set of operators that consume the messages produced from this operator.
    * <p>
    * We use a LinkedHashSet since we need deterministic ordering in initializing/closing operators.
+   *
+   * Note: this is also made transient since the deserialized copies of {@code nextOperatorSpecs} are made externally when
```

Comment:
Comment is somewhat unclear. Where is the DAG _structure_ serialized? Are you saying that this field is set in OperatorImplGraph?

Reply:
Updated the doc.